### PR TITLE
app-misc/ddate: bump to version 0.2.2

### PR DESCRIPTION
**Pull request by ConiKost**
Closes: https://bugs.gentoo.org/667822
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
Package-Manager: Portage-2.3.50, Repoman-2.3.11

### CODE REVIEW

File: app-misc/ddate/ddate-0.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+DESCRIPTION="Perpetual date converter from gregorian to poee calendar"
+HOMEPAGE="https://github.com/bo0ts/ddate"
+SRC_URI="https://github.com/bo0ts/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+LICENSE="public-domain"
+SLOT="0"
+
+RDEPEND="!sys-apps/util-linux[ddate]"
```
Comment (a17r) [inline app-misc/ddate/ddate-0.2.2.ebuild]: blocker should not end up in `DEPEND`, easiest to solve by simply define them in reverse.

**Comment by ConiKost** [comment]
> (would you like to maintain this package?)

Yes, see metadata.xml, added myself, since no maintainer defined? :) Or shouldn't do?

**Comment by a17r** [comment]
> Yes, see metadata.xml, added myself, since no maintainer defined? :) Or shouldn't do?

Ah, I missed it as I did not expect it in the same (and only) commit.

File: app-misc/ddate/ddate-0.2.2.ebuild (hunk 1 above)
Comment (ConiKost) [reply app-misc/ddate/ddate-0.2.2.ebuild]: I am not sure, if that's what you want, by I've removed now 'DEPEND'?

Comment (a17r) [reply app-misc/ddate/ddate-0.2.2.ebuild]: yes, that's also a way to do it.

**Pull request closed without merging**